doc/examples/commctl-user-data-help.txt

```diff
@@ -0,0 +1,28 @@
+usage: commctl user-data [-h] -e ENDPOINT [-c CLUSTER] [-u USERNAME]
+                                  [-p] [-r REMOTE_USER] [-s SSH_KEY_PATH]
+                                  [-a AUTHORIZED_KEYS_PATH] [-C CLOUD_INIT]
+                                  outfile
+
+positional arguments:
+  outfile               Path and file name to output the user-data contents
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -e ENDPOINT, --endpoint ENDPOINT
+                        Commissaire endpoint to use during bootstrapping
+  -c CLUSTER, --cluster CLUSTER
+                        Name of the cluster for new hosts to join
+  -u USERNAME, --username USERNAME
+                        Commissaire user to use when bootstrapping
+  -p, --password        Prompts for a Commissaire password to use when
+                        bootstrapping
+  -r REMOTE_USER, --remote-user REMOTE_USER
+                        Remote user to provide to Commissaire for ssh access
+  -s SSH_KEY_PATH, --ssh-key-path SSH_KEY_PATH
+                        Path to the private key of the remote user
+  -a AUTHORIZED_KEYS_PATH, --authorized-keys-path AUTHORIZED_KEYS_PATH
+                        Path to the authorized_keys file of the remote user
+  -C CLOUD_INIT, --cloud-init CLOUD_INIT
+                        cloud-init.txt file to use
+
+Example: commctl user-data -p -c my_cluster cluster.userdata
```

Comment:
This file is generated by `gendocs.sh`.

Do we typically commit generated documentation like this to source control?  Seems like this could go in a `.gitignore` file.